cva6/bsp/README.md

```diff
@@ -0,0 +1,166 @@
+Board Support Package (BSP) for CV32 Verification
```

Comment:
Should be "BSP for CVA6 Verification"

Reply:
The whole BSP directory is coming from cva6 repository (core/example_tb) without any modifications.
Other files (e.g. Makefile, link.ld) should also be modified to support CVA6.